content/swift/concepts/classes/classes.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Classes'
-Description: 'Classes in Swift defines characteristics and behaviors of an object.'
+Description: 'Classes in Swift define characteristics and behaviors of an object.'
```

Comment:
Should bold first appearance of "Class" below.

Reply:
Sounds good, going to implement this throughout all swift entries by bolding the keyword